frontend/src/views/UploadFileView.tsx

```diff
@@ -31,7 +31,7 @@ const UploadFileView: React.FC = () => {
         onChange={handleFileChange}
         style={{ display: "none" }}
       />
-      <Button label="Upload Data >" onClick={() => { handleButtonClick(); }} />
+      <Button label="get started >" onClick={() => { handleButtonClick(); }} />
```

Comment:
nitpicky but i feel like "Get started >" would be better here, we don't use the lowercase styling anywhere else iirc

Reply:
makes sense!